mbedtls/src/lib.rs

```diff
@@ -163,3 +165,11 @@ pub unsafe extern "C" fn mbedtls_time(tp: *mut time_t) -> time_t {
     }
     timestamp
 }
+
+// Debug not available in SGX
```

Comment:
SGX does not define the MBEDTLS_DEBUG_C needed to have function available.

File: mbedtls-sys/build/config.rs:408
```
    ("std",                   ("MBEDTLS_DEBUG_C",                           Defined)),
```

But this is ignored due to: mbedtls-sys/build/build.rs:43
```
        for &(feat, def) in config::FEATURE_DEFINES {
            if (feat == "std") && (target == "x86_64-fortanix-unknown-sgx") {
                continue;
            }
```

So I'm assuming there was a reason to not have it available. (probably because keys can leak via debug logs if you set it to verbose).

Let me know if I should change this.